### PR TITLE
Create Posts “hello-world”

### PR DESCRIPTION
**Pull request by rafaelferreira6**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/blog/hello-world.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**Finiam** itself already existed as created by a bunch of talented people who started it initially as Onebiller, which also served as the name for their product. Onebiller renamed to **finiam** when they tried to expand their range of products and overgrew the original name. Lastly, the people who brought these two worlds together and empowered a new chapter in **finiam**’s history were the amazing folk at Subvisual, who saw the potential in pivoting the business to a service-focused approach around the Fintech area and brought me and Davide in to make it happen.
+
+This initial merger by two previous teams and with the guidance of Subvisual allowed us to pick up the valuable lessons from each of our histories and forge ahead building something new, evolving and adapting it to fit a new reality and a new challenge.
+
+I think that’s enough looking at the past so let us move forward and I’ll try to explain why we decided to do this and how we do it.
+
+<https://a16z.com/2020/01/21/every-company-will-be-a-fintech-company/>
```
Comment (jfsgomes) [inline src/blog/hello-world.md]:
```suggestion
<iframe width="852" height="479" src="https://www.youtube.com/embed/DjUMfhT0o64" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

**Review by jfsgomes — CHANGES_REQUESTED** [review]

File: src/blog/hello-world.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  memory lane before we get down to business.
+metadata:
+  keywords: fintech, team, founders
+  description: Finiam hit the road running and time truly flew by. As we end 2020,
+    we look back at happened in the first year of the company and the last
+    couple of months and noticed that somewhere along the road we forgot to
+    introduce ourselves. With this we intend to correct that discourtesy but
+    first, bear with me for a little while longer while we make a trip to the
+    memory lane before we get down to business.
+  image: /images/group-74.jpg
```
Comment (jfranciscosousa) [inline src/blog/hello-world.md]: Would be nice to prefix the image with the slug of the blogpost like `hello-world-group-74.jpg`

File: src/blog/hello-world.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+layout: layouts/post.liquid
+tags: post
+title: Hello World
+author: jose-gomes
+category: team
+date: 2021-01-06T12:23:51.596Z
+long_description: Finiam hit the road running and time truly flew by. As we end
+  2020, we look back at happened in the first year of the company and the last
+  couple of months and noticed that somewhere along the road we forgot to
+  introduce ourselves. With this we intend to correct that discourtesy but
```
Comment (jfsgomes) [inline src/blog/hello-world.md]: I would remove `With this we intend to correct that discourtesy but
  first, bear with me for a little while longer while we make a trip to the
  memory lane before we get down to business.` from here

File: src/blog/hello-world.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  first, bear with me for a little while longer while we make a trip to the
+  memory lane before we get down to business.
+metadata:
+  keywords: fintech, team, founders
+  description: Finiam hit the road running and time truly flew by. As we end 2020,
```
Comment (jfsgomes) [inline src/blog/hello-world.md]: I would replace this with `Finiam says hello world :wave:`

**Review by jfsgomes — CHANGES_REQUESTED** [review]

**Pull request merged**